lib/SIL/IR/SILPrinter.cpp

```diff
@@ -77,6 +77,14 @@ llvm::cl::opt<bool>
 SILPrintSourceInfo("sil-print-sourceinfo", llvm::cl::init(false),
                    llvm::cl::desc("Include source annotation in SIL output"));
 
+llvm::cl::opt<bool>
+SILPrintTypes("sil-print-types", llvm::cl::init(false),
+                   llvm::cl::desc("always print type annotations for instruction operands in SIL output"));
+
+llvm::cl::opt<bool>
+SILPrintNoUses("sil-print-no-uses", llvm::cl::init(false),
+                   llvm::cl::desc("omit use comments in SIL output"));
```

Comment:
I know we have a bunch of this already, but we really need to kick the habit of using global `llvm::cl::opt`s instead of breaking down the command line into a context and threading that context into passes. This stuff all turns into global constructors that have to run in every process that even links this code in. The Carbon folks are finding that the uses in the LLVM passes alone are a significant contributor to their compile times; admittedly, they have a much faster compiler overall, but still.

Anyway, just feedback for a later PR.